File: ivis4mac.dataModeling/ivis4mac.entityFrameworkModeling/Ivis4mac/Model/Person.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Ivis4Mac.Model
{
    public class Person
    {
        public int Id { get; set; }
        //ФИО – фамилия, имя и отчество.
        //Фамилия
        //Имя
        //Отчество

        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PatronymicName { get; set; }

        // Дата рождения – выпадающие свитки по числу, месяцу и году, сбоку, 
        // автоматически, в скобках, выставляется сколько полных лет на настоящий момент.
        public DateTime BirthDay { set; get; }
        public Address Address { set; get; }
        public Contact Contact { set; get; }
    }
}

File: app/models/person.rb
class Person
  include DataMapper::Resource
  property :id, Serial
  property :first_name, String, :required => true
  property :last_name,  String, :required => true

  property :second_name, String
  property :patronymic_name, String
  property :birthday, Date
  #
  has n, :signs, :through => Resource


end

class Sign
  include DataMapper::Resource
  property :id, Serial
  belongs_to :sign_value,  :required => true
  belongs_to :sign_type,  :required => true


  has n, :persons, :through => Resource

end

